admin_manual/configuration_server/occ_command.rst

```diff
@@ -1272,14 +1272,16 @@ units.
 Trashbin
 --------
 
-::
+These commands allow for manually managing various aspects of the trash bin (deleted files)::
 
  trashbin
-  trashbin:cleanup  [--all-users] [--] [<user_id>...]  Permanently remove deleted files
-  trashbin:restore  [--all-users] [--scope[=SCOPE]] [--since[=SINCE]] [--until[=UNTIL]] [--dry-run] [--] [<user_id>...]  Restore deleted files according to the given filters
```

Comment:
Having the options is really nice, especially with filters because the GUI is limited in this respect.

Reply:
Currently in the `occ` command docs, in the list of commands for each area the options are never listed. This one was an oddball and it made the table challenging to read. I agree options can make sense in the docs, but not in that spot currently. Many of the command sections have examples and additional bits after the command listing. Perhaps something there in a follow-up PR.